bn-apps/memberships-management/membership-service-contracts-and-states/src/resources/migration/Schema.changelog-Include.xml

```diff
@@ -0,0 +1,4 @@
+<databaseChangeLog>
+    <include file="Schema.changelog-createTable.xml"/>
+    <include file="Schema.changelog-PendingMembershipRequestSchemaV1.xml"/>

```

Comment:
Also can you please rename `Schema.changelog-createTable.xml` to be consistent with the other one?

Reply:
I've made it consistent now